pipelines/respondent-home-ui.yml

```diff
@@ -142,7 +142,7 @@ jobs:
       environment_variables:
         HOST: 0.0.0.0
         LOG_LEVEL: INFO
-        ACCOUNT_SERVICE_URL: https://ohs-alpha.onsdigital.uk
+        ACCOUNT_SERVICE_URL: ((preprod_account_service_url))
```

Comment:
Do we need to leave a comment that this should *exclude* the path prefix [for all RH routes]?